src/Quadrature/quadrature.jl

```diff
@@ -101,9 +108,11 @@ end
 for dim in 2:3
     @eval begin
         function QuadratureRule{RefSimplex{$dim}, T}(quad_type::Symbol, order::Int) where T
-            if $dim == 2 && quad_type === :legendre
+            if $dim == 2 && quad_type === :dunavant && order ≤ 8
```

Comment:
Wouldn't this fail if I put `QuadratureRule{RefTriangle}(9)`?

Reply:
Note that (if I have not messed up anything here when reading through the references) the rule for order > 8 is actually a [Gauss-Jacobi](https://en.wikipedia.org/wiki/Gauss%E2%80%93Jacobi_quadrature) rule and the rule for order < 9 can properly integrate all polynomials on the triangle.